tools/generator/src/Generator/Main.swift

```diff
@@ -41,7 +41,7 @@ extension Generator {
         let buildMode: BuildMode
     }
 
-    static func parseArguments(_ arguments: [String]) throws -> Arguments {
+    static func parseArguments(_: [String]) throws -> Arguments {
         guard CommandLine.arguments.count == 7 else {
```

Comment:
Let's use the passed in value instead.

```suggestion
    static func parseArguments(_ arguments: [String]) throws -> Arguments {
        guard arguments.count == 7 else {
```